### PR TITLE
add ScrollToTop component & fix TnC page

### LINKED ISSUE #98: Website Does Not Scroll to Top on Page switch during navigation.
**Describe the bug**
When navigating to a new page, the website does not scroll to the top of the page. Instead, it stays at the same scroll position from the previous page.

**To Reproduce**

1. Go to the website.
2. Scroll down on any page.
3. Click on any button to navigate to a new page.

4. Observe that the new page does not start at the top.

**Expected behavior**
The page should scroll to the top upon navigation.

**Screen Recording :**

https://github.com/MAVRICK-1/Nest-Ondc/assets/88451512/1fc78add-bc15-4fa2-a716-de5329d1e8b0

### PR DESCRIPTION
**Pull request by kushagra-singhh**
## Description
- Added a ScrollToTop component to the project to ensure the window scrolls to the top whenever a new page is navigated to. 
- Fixed minor bugs in TnC page.
- Fixes and closes https://github.com/MAVRICK-1/Nest-Ondc/issues/98

## How Has This Been Tested?

- Tested by navigating between different routes in the application.
- Tested on local development server.

## Screenrecording : 

https://github.com/MAVRICK-1/Nest-Ondc/assets/88451512/c1fede08-fe53-4ccb-b2a4-02772cc5a115

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -31,6 +29,7 @@ import { collection, doc, getDocs } from "firebase/firestore";
 import MapComponent from "./components/map/ITEMmap";
 import { db } from "./firebase";
 import SellerForm from "./pages/SellerRegistration";
+import ScrollToTop from "./components/ScrollToTop"; // Import the ScrollToTop component
```
Comment (PayalKumari10) [inline src/App.js:32]: Hey @kushagra-singhh 
Under the components directory, create a new folder named ScrollToTop.

**Review by PayalKumari10 — CHANGES_REQUESTED** [review]
Hey @kushagra-singhh 
Under the components directory, create a new folder named ScrollToTop.

File: src/App.js (hunk 1 above)
Comment (PayalKumari10) [inline src/App.js:32]: Hey @kushagra-singhh, Under the components directory, create a new folder named ScrollToTop.

**Review by PayalKumari10 — CHANGES_REQUESTED** [review]
Hey @kushagra-singhh , Under the components directory, create a new folder named ScrollToTop.

File: src/App.js (hunk 1 above)
Comment (akbatra567) [reply src/App.js:32]: Discussed with @PayalKumari10.

Hey @kushagra-singhh, 
Could you convert this component to a helper/util function ? which can be called as and when required to ?

Possible Usecases:
1. If we want to add a button to scrollToTop, we can call the function on click of that button.
2. We can directly call the scrollToTop helper/util function in useEffect of any existing component.

Do add anything if I missed @PayalKumari10.

Comment (PayalKumari10) [reply src/App.js:32]: @akbatra567 , Your approach is correct.

**Comment by kushagra-singhh** [comment]
I'm currently having my exams. I'll make the requested changes after my exams are over.

**Comment by PayalKumari10** [comment]
okay

**Pull request closed without merging**